addons/l10n_ec/models/res_partner.py

```diff
@@ -30,7 +30,9 @@ def get_ats_code_for_partner(cls, partner, move_type):
         Returns ID code for move and partner based on subset of Table 2 of SRI's ATS specification
         """
         partner_id_type = partner._l10n_ec_get_identification_type()
-        if move_type.startswith('in_'):
+        if partner.vat and verify_final_consumer(partner.vat):
```

Comment:
@stevTresCloud @pepetreshere let's separate this into a new PR pointing to V17.0